pkg/clients/aws/route53.go

```diff
@@ -66,7 +67,9 @@ var fedrampHostedZoneID = os.Getenv(fedrampHostedZoneIDVariable)
 
 // awsClient implements the Client interface
 type awsClient struct {
-	client route53iface.Route53API
+	client     route53iface.Route53API
+	kubeClient client.Client
```

Comment:
KubeClient and Namespace are unrelated to AWS, so should not be placed inside the AWS Client